docs/workforce-management-framework/topics/con-workflow-step.adoc

```diff
@@ -0,0 +1,60 @@
+= Workflow Step
+
+== Step Structure
+Steps are divided into to linked catergories the `module` and the `definition`.
+
+=== Module
+Modules can contain one or more angular directives for rendering steps in the portal and mobile applications. 
+{WFM-RC-NameShort} steps use Angular.js directives to build user interface features as part of workflows.
+Each step is divided into two types of directive:
+
+- **Edit Section (Form)**
+
+The Edit section is displayed when workflow is executed and allows users to edit information typically this would be a form.
+The Edit section consists of the html template and directive.
+
+- **Preview Section (View)**
+
+The Preview section is displayed as a summary when a workflow has been completed. The Preview section
+consists of the html template and directive.
+
+=== Definition 
+Definition contains the step definition which outlines the properties such as name and decription of a step for the workflow in the portal application.
```

Comment:
I think it is important to call out the full structure here ( if it's manageable in terms of the 'size' of it - ie: 100 items in the structure is too much)


If the json example below covers all of this, I'm happy to create a table using the details within the json object

Reply:
`decription` of step sounds awesome. 
Structure is bellow.. so let's not mention that at all.

Reply:
I'd prefer to remove the example and add useful reference material, in terms of a table which details the Step definition.
The structure below is an example and as a reader, I was wondering what the full structure is after looking at it.

Reply:
not sure what you mean , the full structure for a single step is in the example

Reply:
Best to ignore my comment at present. I'll hold off making these types of comments until "Tech Ack" has been approved for the PR as I could be causing more confusion.